projects/coordinator/src/main/java/org/batfish/coordinator/ExceptionMapper.java

```diff
@@ -0,0 +1,22 @@
+package org.batfish.coordinator;
+
+import com.google.common.base.Throwables;
+import javax.ws.rs.WebApplicationException;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.ext.Provider;
+
+@Provider
+public class ExceptionMapper implements javax.ws.rs.ext.ExceptionMapper<Exception> {
+  @Override
+  public Response toResponse(Exception exception) {
+    if (exception instanceof WebApplicationException) {
+      // An exception that already maps directly to a standard HTTP Response, likely thrown
+      // somewhere inside Jersey aka if the user typed a bad URI.
+      return ((WebApplicationException) exception).getResponse();
```

Comment:
This actually gives an empty entity body, can use statements like : 
return Response.status(((WebApplicationException) exception).getResponse().getStatus())
          .entity(Throwables.getStackTraceAsString(exception)).build();
to specify an entity body. Also, since our code throws BatfishException everywhere, I think it might be useful to provide handler for BatfishException, use status code other than 500. (maybe BAD_REQUEST? Since most of them are caused by invalid client inputs.)